css/homepage.css

```diff
@@ -0,0 +1,110 @@
+.homepage-content {
+    width: 80%;
+    margin: 3rem auto;    
+}
+
+.homepage-image {
+    width: 100%;
+    border-radius: 0.5rem;
```

Comment:
For something less than 1 rem use pixels ( upto 10px)

Reply:
Okay